<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>659</width>
    <height>533</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Plot Project No :::: 2</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>629</width>
         <height>1496</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="4" column="0" colspan="2">
         <widget class="Line" name="line_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="13" column="0">
         <widget class="QLineEdit" name="lineEdit_4">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(164, 0, 0);
color: rgb(85, 87, 83);
font: 14pt &quot;Ubuntu&quot;;</string>
          </property>
          <property name="text">
           <string>AREA FOR EACH RESOLUTION USED IN THE COMPUTATION</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="2" column="0" colspan="2">
         <widget class="Line" name="line_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="8" column="0" colspan="2">
         <widget class="Line" name="line_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="7" column="0" colspan="2">
         <widget class="QCustomPlot" name="Mismatch_WidgetPlot3" native="true">
          <property name="minimumSize">
           <size>
            <width>606</width>
            <height>321</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="6" column="0" colspan="2">
         <widget class="Line" name="line_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="2">
         <widget class="QCustomPlot" name="Mismatch_WidgetPlot" native="true">
          <property name="minimumSize">
           <size>
            <width>606</width>
            <height>321</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="14" column="0">
         <widget class="Line" name="line_6">
          <property name="minimumSize">
           <size>
            <width>606</width>
            <height>0</height>
           </size>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="3" column="0" colspan="2">
         <widget class="QCustomPlot" name="Mismatch_WidgetPlot2" native="true">
          <property name="minimumSize">
           <size>
            <width>606</width>
            <height>321</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="9" column="0" colspan="2">
         <widget class="QLineEdit" name="lineEdit_3">
          <property name="styleSheet">
           <string notr="true">font: 13pt &quot;Ubuntu Condensed&quot;;
background-color: rgb(32, 74, 135);
color: rgb(237, 212, 0);</string>
          </property>
         </widget>
        </item>
        <item row="11" column="0">
         <widget class="QCustomPlot" name="Mismatch_WidgetPlot4" native="true">
          <property name="minimumSize">
           <size>
            <width>606</width>
            <height>321</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="10" column="0" colspan="2">
         <widget class="Line" name="line">
          <property name="minimumSize">
           <size>
            <width>321</width>
            <height>0</height>
           </size>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="5" column="0" colspan="2">
         <widget class="QLineEdit" name="lineEdit_2">
          <property name="styleSheet">
           <string notr="true">font: 13pt &quot;Ubuntu Condensed&quot;;
background-color: rgb(32, 74, 135);
color: rgb(237, 212, 0);</string>
          </property>
         </widget>
        </item>
        <item row="12" column="0">
         <widget class="Line" name="line_7">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QLineEdit" name="lineEdit">
          <property name="styleSheet">
           <string notr="true">font: 13pt &quot;Ubuntu Condensed&quot;;
background-color: rgb(32, 74, 135);
color: rgb(237, 212, 0);</string>
          </property>
          <property name="text">
           <string>SURFACE : </string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>659</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
